gem/ci/build.ps1

```diff
@@ -17,8 +17,10 @@ function Get-ContainerVersion
         git fetch origin 'refs/tags/*:refs/tags/*'
     }
 
-    $gitver = git describe
-    if ($LASTEXITCODE -ne 0) { return '0.0.0' }
+    # in a repo with no tags, --always returns a 8 character SHA
+    $gitver = git describe --always
+    # with no dashes, this is just a SHA, so fall back to 0 version
```

Comment:
Actually.....wondering if it might be good to use the SHA as a tag?

Not that we'd actually publish that container, but at least we'd know what state the repo was in when it was built.

Reply:
Can we use that as a Docker version?

If so, that seems fine to me too

Reply:
Yeah, just tried it, works. 

I haven't seen any limitations to what the image tag can be, seems like I've seen all sorts of variants.

Reply:
I liked your idea better... it's simpler to just use the actual `git describe` always